lib/middleware/with-telemetry.ts

```diff
@@ -36,6 +37,19 @@ const sendReport = async (message: string, report: Report) => {
   }
 }
 
+const getRequestBody = async (request: Request) => {
+  if (!request.body) {
+    return
+  }
+
+  try {
+    return await request.json()
+    // eslint-disable-next-line unused-imports/no-unused-vars
```

Comment:
https://github.com/ethproofs/ethproofs/blob/f52bec2606ff9c66e726d93315529b5291ce5488/.eslintrc.json#L59

We fixed the linting rule here so we shouldn't need this anymore

```suggestion
```